### PR TITLE
Init the base box

### PR DESCRIPTION
**Pull request by Almad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,52 @@
-docker-base-images
-==================
+# Apiary's base Docker images
 
-Base docker images for Apiary applications
+This repository helps to keep Apiary environment, services and libraries consistent and in sync. 
+
+Each directory have area of responsibility and the libraries should derive from the respective image.
```
Comment (honzajavorek) [inline README.md]: has an area?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,52 @@
...
+
+As a lot of applications/libraries are deployed on Heroku, images are usually derived from the Ubuntu version from the latest Heroku stack. Libraries use what's the minimal stack available.
+
+## Naming
+
+Base development images uses `base-dev-` prefix.
```
Comment (honzajavorek) [inline README.md]: use?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,52 @@
...
+
+Base deployment/runtime images uses `base-` prefix.
```
Comment (honzajavorek) [inline README.md]: use?

File: README.md [hunk 4]
```diff
@@ -1,4 +1,52 @@
...
+
+## Versioning
+
+We keep `latest` tag for testing and that is updated at-will. Otherwise, semver is used:
+
+* Arch is reserved for OS changes/upgrades (Ubuntu LTS change, ...)
+* Major is reserved for packages being added or deleled
```
Comment (honzajavorek) [inline README.md]: deleled

File: README.md [hunk 5]
```diff
@@ -1,4 +1,52 @@
...
+
+## Naming
+
+Base development images uses `base-dev-` prefix.
+
+Base deployment/runtime images uses `base-` prefix.
+
+## Versioning
+
+We keep `latest` tag for testing and that is updated at-will. Otherwise, semver is used:
```
Comment (honzajavorek) [inline README.md]: If `semver` is used, use commonly recognized terms... this is the first time in my life I read about `arch` version. Semver uses `MAJOR.MINOR.PATCH`.

Also, semver could be hyperlinked to the website.

Comment (Almad) [inline README.md]: OK, I am used to `arch.major.minor-patch` :)

Comment (Almad) [inline README.md]: @honzajavorek what is the difference between "linked" and "hyperlinked" (unless you are hypermedia junkie)? :)

Comment (honzajavorek) [inline README.md]: @Almad the same difference as between "bike" and "bicycle". None.

Comment (Almad) [inline README.md]: @honzajavorek I thought "bike" and "bikeshed" :))

But OK, fair points—should keep language same with semver.

**Pull request merged**